Felienne/hedy
# [BUG FIX] Adds accidentally removed keyword language select on signup

**Description**
Due to the merging of #2024 we accidentally deleted the earlier implemented option select for a keyword language. In this PR we re-implement and improve this option. The keyword select option is only shown if a non-english keyword supported language is selected. This is done dynamically on language input change. All other keyword languages expect English and the selected language are hidden. When the select language does not support localized keywords the option is hidden and English is automatically selected. We also add a small fix where the keyword language for accounts created through multiple-account creation is automatically set to the teacher language.

**Note**
This dynamic showing/hiding of options does **not** work on Safari. If the user selects a keyword language that is neither their own language or English we return an error from the back-end. I think this is fine for now.

**Fixes**
No related issue.

**How to test**
Navigate to the sign-up page and select a keyword-supported non-english language. Notice the new select option that shows and verify that it works as expected. Also verify that (when on Safari) an error occurs if a non-english non-selected language is chosen. In any other browser only the two valid options (English and the select language) are shown.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
